doc/nrf/known_issues.rst

```diff
@@ -1018,7 +1006,7 @@ NCSDK-6832: SMP Server sample fails upon initialization
 SEGGER Embedded Studio Nordic Edition
 *************************************
 
-.. rst-class:: v1-4-2 v1-4-1 v1-4-0
+.. rst-class:: v1-5-0 v1-4-2 v1-4-1 v1-4-0
```

Comment:
The issue below is only valid for SES 5.10d, which is the NCS for 1.4.x.
NCS 1.5.0 ships with SES 5.34a, where this issue is not present.

Reply:
We can handle it in my subsequent known-issues PR. Talk to me @tejlmand (and remove change request) so we can get this PR in